manim/animation/indication.py

```diff
@@ -74,7 +74,7 @@ class Flash(AnimationGroup):
         "run_time": 1,
     }
 
-    def __init__(self, point, color=YELLOW, **kwargs):
+    def __init__(self, point, color=Colors.yellow.value, **kwargs):
         self.point = point
         self.color = color
```

Comment:
Stuff like this can be changed to the following:

```suggestion
        self.color = hex(color.value) if ((color in Colors) or isinstance(color,Enum)) and type(color.value) is int
```

We can then have the colors defined in hex as such:
```py
class Colors(Enum):
    white = 0xff000000 ## 0x<alpha><R><G><B>, will get converted to an int but we can just get the hex value when needed.
```

Some changes would be required in `utils/color.py` for this to work.

Reply:
Yeah so instead of littering the codebase with this kind of code, I think we should do one of the following:
1. only use one color format throughout the codebase so there's no need to do input validation/conversion.

2. use a decorator that performs input validation.

3. have our own Color class, that inherits from colour.Color.

I think any of these will be better than copying the line `hex(color.value) if ((color in Colors) or isinstance(color,Enum)) and type(color.value) is int` all over the codebase. What do you think?